src/components/ui/Sidebar/Folder.tsx

```diff
@@ -59,8 +59,8 @@ export default function Folder({ folderFavorite, setOpenZones }: FolderProps) {
         'GET',
         cookies['_xsrf']
       );
-      const data = await response.json();
-      if (data.status === 200) {
+
+      if (response.status === 200) {
```

Comment:
@krokicki if this is all we need here - shouldn't there be a HEAD endpoint?

Reply:
Great idea. I'll add it to the issue tracker and we can come back to it later.